src/ui/ChatWindow.ts

```diff
@@ -83,7 +84,65 @@ export default class ChatWindow {
       this.element.attr('data-presence', Presence[this.contact.getPresence()]);
       this.element.attr('data-subscription', this.contact.getSubscription());
 
-      this.getAccount().triggerChatWindowInitializedHook(this, contact);
+      this.getAccount().triggerChatWindowInitializedHook(this, contact);   this.element.find('div.jsxc-message-area').on('contextmenu',{ref:this},this.openContextMenu);
+
+      $(document).on('mousedown', function (e) {
```

Comment:
Is this handler removed somewhere?

Reply:
now removed on close method of the class

Reply:
ok the mousedown handler is now bounded to the chatwindow only so i can remove the handler in the close method